### DISCUSSION TITLE
🔥 技术&开源&文章推荐

### DISCUSSION THREAD

**Discussion by ATQQ** [ATQQ/sugar-blog#123] [Ideas]
欢迎大家分享&推荐🙏🏻

**Comment by Perfecto23** [comment]
我觉得这个不错：
https://mp.weixin.qq.com/s?__biz=MzA4Njc2MTE3Ng==&mid=2456151975&idx=1&sn=7d81ec04635396abeafdf7f723a09501&scene=21#wechat_redirect

**Reply by ATQQ** [reply]
我下来看看这个好不好上手😂，社区有一个tRPC

**Comment by Perfecto23** [comment]
https://mp.weixin.qq.com/s?__biz=MzIzOTU0NTQ0MA==&mid=2247534374&idx=1&sn=d4fdd79935f4844f57e058cffb67cfc8&chksm=e92a7a29de5df33fed3c5de533c72fb95aa65dfa5e3490e876eff9ae275b840fc2026e70466d&mpshare=1&scene=1&srcid=08110cciaE8QHsjzLSKisPrd&sharer_sharetime=1691717013344&sharer_shareid=31c55da0340daae091e3d76b40a7b2dc&from=industrynews&version=4.1.8.6020&platform=win#rd
